src/lib/validate-input.js

```diff
@@ -33,6 +33,40 @@ export function validateBatchNormalizationParams(input, mean, variance,
   check1DTensorWithSize(bias, dim, 'bias');
 }
 
+export function validateLayerNormalizationParams(input, {axes, scale, bias} = {}) {
+  if (scale && axes ) {
+    if (scale.rank !== axes.length) {
+      throw new Error('DataError: the rank of scale is not equal to the size of axes.');
```

Comment:
These error messages are totally following **Algorithm** part of [layerNormalization](https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/pull/478.html#api-mlgraphbuilder-layernorm) op, it would be todo enhancement to update validation checking for others ops.

Reply:
sounds good to me, thanks!